bionic/protocols.py

```diff
@@ -197,6 +198,39 @@ def __repr__(self):
         return f"{self.__class__.__name__}(...)"
 
 
+class JsonProtocol(BaseProtocol):
+    """
+    Decorator indicating that an entity's values can be serialized to the
+    ``JSON`` format.
```

Comment:
I think it makes sense to omit the backquotes here, or even just say "can be serialized to JSON". The backquotes make this render like code, which it isn't.